### PR TITLE
Feature/scan qrcode screen

### PR DESCRIPTION
**Pull request by sakshisharma1999**

### CODE REVIEW

**Review by dewale005 — CHANGES_REQUESTED** [review]
you would need to add some changes to this 

1.  you need to create all the components interface file and define the props
2. you would need to create a file the holds the app content and let the static content be where your content is defined from
3. remove all inline styles

File: src/screens/tpn.verify.screen.tsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+} from 'react-native-safe-area-context';
+
+const TpnVerifyScreen: React.FC<any> = () => {
+  const navigation =
+    useNavigation<NativeStackNavigationProp<RootStackParamList>>();
+  const insets = useSafeAreaInsets();
+  return (
+    <SafeAreaView
+      edges={['left', 'right']}
+      style={{ flex: 1, backgroundColor: 'white' }}>
```
Comment (dewale005) [inline src/screens/tpn.verify.screen.tsx:21]: there are inline styles here

File: src/screens/tpn.voucher.screen.tsx [hunk 2]
```diff
@@ -0,0 +1,226 @@
...
+
+const TpnVoucherScreen: React.FC<any> = () => {
+  const navigation =
+    useNavigation<NativeStackNavigationProp<RootStackParamList>>();
+
+  const insets = useSafeAreaInsets();
+  return (
+    <SafeAreaView
+      edges={['left', 'right']}
+      style={{ flex: 1, backgroundColor: 'white' }}>
```
Comment (dewale005) [inline src/screens/tpn.voucher.screen.tsx:33]: remove inline styles

**Review by dewale005 — CHANGES_REQUESTED** [review]

File: src/screens/tpn.verify.screen.tsx (hunk 1 above)
Comment (sakshisharma1999) [reply src/screens/tpn.verify.screen.tsx]: changes done. Please check now.

File: src/screens/tpn.voucher.screen.tsx (hunk 2 above)
Comment (sakshisharma1999) [reply src/screens/tpn.voucher.screen.tsx]: changes done. Please check now.

**Pull request merged**